stdlib/public/core/Mirror.swift

```diff
@@ -705,3 +707,77 @@ extension Mirror: CustomReflectable {
     return Mirror(self, children: [:])
   }
 }
+
+#else  // SWIFT_ENABLE_REFLECTION
+
+@available(*, unavailable)
```

Comment:
Would it be possible to only `#if-else` the `@availability` annotation instead of duplicating all of the Mirror declarations here?